### PR TITLE
added hw_05

### PR DESCRIPTION
**Pull request by SPrudnikova**

### CODE REVIEW

File: hw_05/Program.cs [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+            Console.WriteLine($"Enter {arrSize} integer to fill an array");
+            int[] userArr = FillArrayByUser(new int[arrSize], 0, arrSize);
+
+            int[] resultArr = new int[arrSize];
+            for (int i = 0; i < arrSize; i++)
+            {
+                resultArr[i] = userArr[i] + randomArr[i];
+            }
+
```
Comment (VadzimPapko) [inline hw_05/Program.cs:34]: I would also make a separate method to fill the result array out like you have done for both cases above.

File: hw_05/Program.cs [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            Console.WriteLine("Initial array:");
+            PrintArray(randomArr);
+            Console.WriteLine();
+
+            Reverse(randomArr);
+
+            Console.WriteLine("Result array:");
+            PrintArray(randomArr);
+
```
Comment (VadzimPapko) [inline hw_05/Program.cs]: please delete unused empty line there

File: hw_05/Program.cs [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+            Random randomGenerator = new Random();
+            for (int i = from; i < to; i++)
+            {
+                arr[i] = randomGenerator.Next(0, 100);
+            }
+
+            return arr;
+        }
+
+        public static T[] Reverse<T>(T[] arr)
```
Comment (VadzimPapko) [inline hw_05/Program.cs:146]: I see you have been using generic Type there. That is great! Have you discovered this approach yourself?

**Review by VadzimPapko — APPROVED** [review]

File: hw_05/Program.cs (hunk 3 above)
Comment (SPrudnikova) [reply hw_05/Program.cs:146]: I use generic  in TypeScript, it's very similar as i found out

Comment (VadzimPapko) [reply hw_05/Program.cs:146]: sure, this feature was derived from C#. 
all right!

File: hw_05/Program.cs (hunk 2 above)
Comment (SPrudnikova) [reply hw_05/Program.cs]: done

File: hw_05/Program.cs (hunk 1 above)
Comment (SPrudnikova) [reply hw_05/Program.cs:34]: done

**Pull request merged**